tests/integration/blackbox_test.py

```diff
@@ -356,7 +376,15 @@ def test_app_with_report(filebase: str):
     )
 
     # 2. Run the requests to obtain sequence of Dryrun responses:
-    dryrun_results = Executor.dryrun_app_on_sequence(algod, teal, inputs)  # type: ignore
+    accounts = [
+        Account(
+            address=get_application_address(Executor.EXISTING_APP_CALL),
+            status="Online",
+            amount=105000000,
+            amount_without_pending_rewards=10500000,
+        )
+    ]
+    dryrun_results = Executor.dryrun_app_on_sequence(algod, teal, inputs, dryrun_accounts=accounts)  # type: ignore
```

Comment:
Do you still want to test with `Account()` even though none of the `APP_SCENARIOS` require it?

Reply:
resolving as I've removed `Account()` from this test

Reply:
resolving as adding an error message assertion